server/ops/docker/build-static-svgbob

```diff
@@ -1,13 +1,4 @@
 # build static executable binary
 FROM ekidd/rust-musl-builder
 
-RUN git clone https://github.com/ivanceras/svgbob.git /home/rust/src
-
-RUN sudo chown -R rust:rust .
-
-WORKDIR /home/rust/src/svgbob_cli
-
-# version 0.4.1
-RUN git checkout 43fb0364e989d0e9a7656b148c947d47cc769622
-
-RUN cargo build --release
+RUN cargo install --version 0.4.2 svgbob_cli
```

Comment:
We should probably update the version to 0.4.2 in: 

- https://github.com/yuzutech/kroki/blob/71f661a64c8e7424d3ff59a155494fbb18a0e193/Makefile#L23 
- https://github.com/yuzutech/kroki/blob/71f661a64c8e7424d3ff59a155494fbb18a0e193/server/src/main/java/io/kroki/server/service/HelloHandler.java#L41

And in the documentation: [architecture.adoc#gateway-server](https://github.com/yuzutech/kroki/blob/71f661a64c8e7424d3ff59a155494fbb18a0e193/docs/modules/ROOT/pages/architecture.adoc#gateway-server).